sdk/typescript/runtime/main.go

```diff
@@ -218,23 +177,125 @@ func (t *TypescriptSdk) Base(runtime SupportedTSRuntime) (*Container, error) {
 		return dag.Container().
 			From(nodeImageRef).
 			WithoutEntrypoint().
+			WithExec([]string{"corepack", "enable"}).
```

Comment:
Would be good to get a comment for what this does.

Reply:
Done in [387eb49](https://github.com/dagger/dagger/pull/7096/commits/387eb49a0da4916c410830d8cbc6a72cd086dde2)